File: back/ProEventos.Domain/Palestrante.cs
using System.Collections.Generic;

namespace ProEventos.Domain
{
    public class Palestrante
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Telefone { get; set; }
        public List<RedeSocial> RedeSocials { get; set; }

        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}

File: front/ProEventos-App/src/app/models/Palestrante.ts
import { Evento } from "./Evento";
import { RedeSocial } from "./RedeSocial";

export interface Palestrante {
    id:number;
    nome:string;
    email:string;
    telefone:string;
    redesocials: RedeSocial[];
    palestranteseventos:Evento[];
}
